### PR TITLE
Fix uploader object bug, relocate to voiceItem, and pass context

### PR DESCRIPTION
**Pull request by TimRexJeff**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: voicer-client/src/components/audio/AudioUploader.jsx [hunk 1]
```diff
@@ -25,9 +30,11 @@ const ReactUploadMedia = () => {
 		axiosWithAuth()
 		.post("/api/voice", formData, config)
 			.then(res => {
-				alert("File successfully uploaded")
+				refreshAppHandler()
+				// alert("File successfully uploaded")
```
Comment (alphaseinor) [inline voicer-client/src/components/audio/AudioUploader.jsx:34]: smart, brought the refresh so the page will show the new item

Comment (j0liver) [reply voicer-client/src/components/audio/AudioUploader.jsx:34]: Clean Code!

File: voicer-client/src/components/audio/AudioUploader.jsx [hunk 2]
```diff
@@ -84,4 +91,4 @@ function RegFields(props) {
 	  )
 }
 
-export default ReactUploadMedia
\ No newline at end of file
+export default AudioUploader
```
Comment (alphaseinor) [inline voicer-client/src/components/audio/AudioUploader.jsx:94]: no more being made fun of for having a non semantic name

File: voicer-client/src/components/voice/EditProfile.jsx [hunk 3]
```diff
@@ -82,7 +81,6 @@ export default function EditProfile({ profileData, token }) {
             SUPER SUBMIT
           </Button>
```
Comment (alphaseinor) [inline voicer-client/src/components/voice/EditProfile.jsx:82]: can we just say submit edit?

File: voicer-client/src/components/voice/VoiceItem.jsx [hunk 4]
```diff
@@ -70,6 +71,7 @@ const VoiceItem = ({ data, token, bio, currentDisplayName }) => {
           {data.samples[0] !== undefined && (
             <AudioPlayer samples={data.samples} />
           )}
+          {crud && (<AudioUploader />)}
```
Comment (alphaseinor) [inline voicer-client/src/components/voice/VoiceItem.jsx:74]: much better location for the uploader

**Pull request merged**